create-rust-app_cli/src/main.rs

```diff
@@ -10,18 +10,17 @@ use std::path::PathBuf;
 use structopt::StructOpt;
 
 use crate::project::CreationOptions;
-use console::Term;
 use content::project;
-use dialoguer::{theme::ColorfulTheme, Input, MultiSelect, Select};
+use dialoguer::{console::Term, theme::ColorfulTheme, Input, MultiSelect, Select};
```

Comment:
Fixed issues caused by updating console from 0.14.1 to 0.15.5

Don't see why this would cause issues, but it might so I'd like a second look @Wulf